lib/jsonapi/pagination.rb

```diff
@@ -16,7 +16,10 @@ def jsonapi_paginate(resources)
       if resources.respond_to?(:offset)
         resources = resources.offset(offset).limit(limit)
       else
-        resources = resources[(offset)..(offset + limit)]
+        original_size = resources.size
+        resources = resources[(offset)..(offset + limit - 1)]
```

Comment:
I personally think just doing `resources[(offset)...(offset + limit)]` would be more elegant (so 3 dots instead of 2); but this is fine as well.

Reply:
Thanks @jf, I'll keep this to have a better visual differentiator. I guess the lack of one is the main reason what brought us here :see_no_evil:

Reply:
Yeah, that's fine. Thanks for the fix!